girder_worker/app.py

```diff
@@ -84,6 +85,11 @@ def gw_task_prerun(task=None, sender=None, task_id=None,
 
         task.job_manager = deserialize_job_info_spec(**jobSpec)
 
+        # Trim off the /job/jobId
```

Comment:
Ideally the `jobInfoSpec` could pass the actual `workerApiUrl()` output as a separate key.